config/network-policy/allow-webhook-traffic.yaml

```diff
@@ -1,26 +1,31 @@
 # This NetworkPolicy allows ingress traffic to your webhook server running
 # as part of the controller-manager from specific namespaces and pods. CR(s) which uses webhooks
-# will only work when applied in namespaces labeled with 'webhook: enabled'
+# will only work when applied in namespaces labeled with 'webhook: enabled' or OpenShift/Kube APIServer
 apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
   labels:
     app.kubernetes.io/name: odh-model-controller
     app.kubernetes.io/managed-by: kustomize
-  name: odh-model-controller # Original scaffolded name is allow-webhook-traffic
-  namespace: system
+  name: odh-model-controller
+  namespace: opendatahub
 spec:
   podSelector:
     matchLabels:
       control-plane: odh-model-controller
   policyTypes:
     - Ingress
   ingress:
-    # This allows ingress traffic from any namespace with the label webhook: enabled
     - from:
       - namespaceSelector:
           matchLabels:
-            webhook: enabled # Only from namespaces with this label
+            webhook: enabled
+      - namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: openshift-kube-apiserver
+      - namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: openshift-apiserver
```

Comment:
```suggestion
      - namespaceSelector:
          matchExpressions:
            - key: policy-group.network.openshift.io/host-network
              operator: Exists
```

Reply:
This is a solution for a specific environment, I wonder if we need this in the "default settings" or should this be just documented? https://access.redhat.com/solutions/7008681


@israel-hdez where can I learn about "webhook: enabled" label? It's the first time I see this solution.